controllers/replication.storage/volumereplication_controller.go

```diff
@@ -363,13 +372,34 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 
 	instance.Status.LastCompletionTime = getCurrentTime()
+
+	var requeueForInfo bool
+	if instance.Spec.ReplicationState == replicationv1alpha1.Primary {
+		lastSyncTime, err := r.getVolumeReplicationInfo(vr)
+		if err != nil {
+			logger.Error(err, "Failed to get volume replication info")
+		}
+		lastSyncTime1 := lastSyncTime.AsTime()
```

Comment:
if this can not fail, just place it in `metav1Time := metav1.NewTime(lastSyncTime.AsTime())`.

`metav1Time` sounds more like a package name, similar to `metav1`. This is just a variable, so why not call it `time`?